maat-court-data-api/src/main/java/gov/uk/courtdata/prosecutionconcluded/service/ProsecutionConcludedService.java

```diff
@@ -82,14 +86,24 @@ private void processOutcome(ProsecutionConcluded prosecutionConcluded, WQHearing
         prosecutionConcludedImpl.execute(concludedDTO);
     }
 
-
     private WQHearingEntity getWqHearingEntity(ProsecutionConcluded prosecutionConcluded) {
         List<WQHearingEntity> wqHearingEntityList = wqHearingRepository
                 .findByMaatIdAndHearingUUID(prosecutionConcluded.getMaatId(), prosecutionConcluded.getHearingIdWhereChangeOccurred().toString());
         if (wqHearingEntityList.isEmpty() && prosecutionConcluded.isConcluded()) {
             prosecutionConcludedDataService.execute(prosecutionConcluded);
+            triggerHearingDataProcessing(prosecutionConcluded);
```

Comment:
The hearing info may not be available at this point. I would have thought it would be better if we could do that as part of the schedulers.

Reply:
This is now called as part of the scheduler.